packages/schematics/update/update/index_spec.ts

```diff
@@ -153,4 +153,37 @@ describe('@schematics/update', () => {
       }),
     ).subscribe(undefined, done.fail, done);
   });
+
+  it('can migrate from only (short version number)', done => {
+    // Add the basic migration package.
+    const content = virtualFs.fileBufferToString(host.sync.read(normalize('/package.json')));
+    const packageJson = JSON.parse(content);
+    packageJson['dependencies']['@angular-devkit-tests/update-migrations'] = '1.6.0';
+    host.sync.write(
+      normalize('/package.json'),
+      virtualFs.stringToFileBuffer(JSON.stringify(packageJson)),
+    );
+
+    schematicRunner.runSchematicAsync('update', {
+      packages: ['@angular-devkit-tests/update-migrations'],
+      migrateOnly: true,
+      from: '0',
+    }, appTree).pipe(
+      map(tree => {
+        const packageJson = JSON.parse(tree.readContent('/package.json'));
+        expect(packageJson['dependencies']['@angular-devkit-tests/update-migrations'])
+          .toBe('1.6.0');
+
+        // Check install task.
+        expect(schematicRunner.tasks).toEqual([
+          {
+            name: 'run-schematic',
+            options: jasmine.objectContaining({
+              name: 'migrate',
```

Comment:
I think this should be checking the install task, like some of the other tests in this file.

Reply:
There's no installation in a migration only.